### DISCUSSION TITLE
OpenAPI Mocking using body as array and reuse inside response

### DISCUSSION THREAD

**Discussion by AteDev** [microcks/microcks#749] [Q&A] [answered]
Hello, i'm facing an issue when using body array, see example below

Openapi
[test.yaml.txt](https://github.com/microcks/microcks/files/10664736/test.yaml.txt)

http://localhost:8080/rest/Test/1.0/resources/v1/test with POST method

body as json content-type
`[{"foo":{"bar":123},"quantity":1}]`

result : 
`{
    "status": "OK",
    "items": [
        {
            "status": "OK",
            "request": {
                "body": {
                    "foo": {
                        "bar": "null"
                    },
                    "quantity": "null"
                },
                "response": {
                    "httpStatus": 200,
                    "body": {
                        "foo": {
                            "bar": "null"
                        },
                        "date": "2023/02/06"
                    }
                }
            }
        }
    ]
}`

i was unable to reuse body value, what did i do wrong ?

**Comment by lbroudoux** [comment]
Thanks for your message! You did nothing wrong here... seems like a bug as I was able to reproduce it.

**Comment by lbroudoux** [comment, marked as answer]
Oh I just saw that you created the issue #750 to track it. Excellent !

**Reply by lbroudoux** [reply]
Now fixed in #750 !
